src/models/jira-client.ts

```diff
@@ -52,4 +52,15 @@ export class JiraClient {
 	async appPropertiesDelete() {
 		return await this.axios.delete(`/rest/atlassian-connect/latest/addons/${getAppKey()}/properties/is-configured`);
 	}
+
+	async checkAdminPermissions(accountId: string) {
```

Comment:
Going off Gary's comment above https://github.com/atlassian/github-for-jira/pull/2030/files#r1157920279, if it turns out sub isn't present because the user isn't logged in, are we expecting an empty string? Or would it be null/undefined? If it's either of the latter we should probably update the expected type to reflect this. Or better yet, add a check before we even call `checkAdminPermissions`

Reply:
Good call!! have returned early, no userId, no care, have added test case too 👍 ⚡